### PR TITLE
feat(content-search-toolkit): Tune query and allow fuzzy

### PR DESCRIPTION
**Pull request by rafnarnason**
this should improve search results overall to reflect popular content and forgive typos

Attach a link to issue if relevant

## What

Specify what you're trying to achieve

## Why

Specify why you need to achieve this

## Screenshots / Gifs

Attach Screenshots / Gifs to help reviewers understand the scope of the pull request

## Checklist:

- [x] I have performed a self-review of my own code
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] Formatting passes locally with my changes
- [ ] I have rebased against main before asking for a review

### CODE REVIEW

**Review by kksteini — APPROVED** [review]

File: libs/content-search-toolkit/src/queries/search.ts [hunk 1]
```diff
@@ -102,14 +109,29 @@ export const searchQuery = (
...
+              modifier: 'log1p',
+              missing: 1,
+            },
+          },
+          { filter: { range: { processEntryCount: { gte: 1 } } }, weight: 2 },
+        ],
       },
     },
-    ...(Object.keys(aggregation.aggs).length ? aggregation : {}), // spread aggregations if we have any
+    ...(Object.keys(aggregation.aggs).length ? aggregation : {}), // spread aggregations if we have any,
```
Comment (RunarVestmann) [inline libs/content-search-toolkit/src/queries/search.ts:134]: Perhaps the comma in the comment is unintentional :P

File: libs/content-search-toolkit/src/queries/search.ts [hunk 2]
```diff
@@ -34,11 +34,18 @@ export const searchQuery = (
   let minimumShouldMatch = 1
 
   should.push({
-    simple_query_string: {
+    multi_match: {
+      fields: [
+        'title^6', // note boosting ..
+        'title.stemmed^2', // note boosting ..
```
Comment (RunarVestmann) [inline libs/content-search-toolkit/src/queries/search.ts:40]: Not sure what you mean by note boosting ..

Do you mean that the comment is a note implying that the change is boosting results?

Or is this a leftover comment? I'm just wondering :D

**Review by RunarVestmann — APPROVED** [review]
Looks good to me, just wondering about comments

File: libs/content-search-toolkit/src/queries/search.ts (hunk 2 above)
Comment (rafnarnason) [reply libs/content-search-toolkit/src/queries/search.ts:40]: example the "^6" part of "title^6" is giving the fields different weight/boosting ... just added the comments so the reader will note that

File: libs/content-search-toolkit/src/queries/search.ts (hunk 1 above)
Comment (rafnarnason) [reply libs/content-search-toolkit/src/queries/search.ts]: well spotted ... fixed

File: libs/content-search-toolkit/src/queries/search.ts (hunk 2 above)
Comment (RunarVestmann) [reply libs/content-search-toolkit/src/queries/search.ts:40]: Ok cool 👍

**Pull request merged**